### DISCUSSION TITLE
Clickhouse Client depends on System.Net.Http.Formatting

### DISCUSSION THREAD

**Discussion by eugenpaval** [DarkWanderer/ClickHouse.Client#500] [General]
Recently upgraded my web API to .NET9 in preparation to its release in 3 months. Suddenly the build process complains about a few packages the client depends on as being restored for .NET Framework (System.Net.Http.xxx, System.text.Json,...). While the API works on Windows I wonder if this would create issues on a Linux box where this API ultimately runs in Production.

**Comment by MaceWindu** [comment]
Aslready fixed #497
